templates/server/puppetserver/conf.d/webserver.conf.erb

```diff
@@ -0,0 +1,15 @@
+webserver: {
+    access-log-config = <%= scope.lookupvar('puppet::server_puppetserver_dir') %>/request-logging.xml
+    client-auth       = want
+    ssl-host          = 0.0.0.0
+    ssl-port          = <%= scope.lookupvar('puppet::server_port') %>
+    ssl-cert          = <%= scope.lookupvar('puppet::server_ssl_dir') %>/certs/<%= @fqdn.downcase %>.pem
+    ssl-key           = <%= scope.lookupvar('puppet::server_ssl_dir') %>/private_keys/<%= @fqdn.downcase %>.pem
+<%- if scope.lookupvar('puppet::server_ca') -%>
+    ssl-ca-cert       = <%= scope.lookupvar('puppet::server_ssl_dir') %>/ca/ca_crt.pem
+    ssl-cert-chain    = <%= scope.lookupvar('puppet::server_ssl_dir') %>/ca/ca_crt.pem
+    ssl-crl-path      = <%= scope.lookupvar('puppet::server_ssl_dir') %>/ca/ca_crl.pem
+<%- else -%>
+    ssl-ca-cert       = <%= scope.lookupvar('puppet::server_ssl_dir') %>/certs/ca.pem
```

Comment:
Can't we reuse what's defined in ```puppet::server```? See https://github.com/theforeman/puppet-puppet/blob/26e1213d0df7e2efc16329616b620c94494da251/manifests/server.pp#L6-L14